cluster/src/main/java/com/hartwig/pipeline/calling/structural/StructuralCaller.java

```diff
@@ -77,18 +81,27 @@ public String namespace() {
         commands.add(new ExportVariableCommand("PATH",
                 format("${PATH}:%s", dirname(new VersionedToolCommand("samtools", "samtools", Versions.SAMTOOLS).asBash()))));
 
+        // TEMP
```

Comment:
This necessary still? The resources in the image are static we should just be able to set this up exactly how we want it and remove the need for checking files and copying.